swim/memberlist.go

```diff
@@ -280,6 +305,17 @@ func (m *memberlist) Update(changes []Change) (applied []Change) {
 	if len(applied) > 0 {
 		oldChecksum := m.Checksum()
 		m.ComputeChecksum()
+
+		for _, change := range applied {
+			if change.Source != m.node.address {
+				m.logger.WithFields(bark.Fields{
+					"remote": change.Source,
+					// changes in ringpop go do not have an id.
+					// "updateId": change.id,
```

Comment:
Good question.

In the shared integration tests between node and go we have marked the `id` on a change as optional so it works on both implementations. So it would be great to add an `id` to go and make it a required field in the integration tests.

That said. This id is not used _anywhere_ so it add little value. With this I think it will not be added soon to ringpop-go.

I will remove this 'code smell' :)